Remora.Discord.Commands/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -154,7 +154,26 @@ public static IServiceCollection AddInteractionResponder
             this IServiceCollection serviceCollection
         )
         {
+            return serviceCollection.AddInteractionResponder(null);
```

Comment:
As in, just adding a defaulted parameter to the existing method? I figured a non-breaking change was better. I can change it if you don't care.

Reply:
I'm mainly concerned about source-level breakages, since practically nobody swaps out libraries without also recompiling their project these days. You have a good point, though, so either works for me.

Reply:
What I can do is go ahead and queue up a PR to combine them, as soon as this one gets pulled, and you can just save that until the next time you're making a major version change anyway.

Reply:
That sounds good :)